packages/design-system/src/components/web-components/ds-alert.ts

```diff
@@ -15,7 +15,6 @@ const attributes = [
   'analytics-parent-heading',
   'analytics-parent-type',
   'id',
-  'alert-ref',
```

Comment:
You know, maybe we could create some special element object interface in the future to expose certain elements without having the user query for them. It'd be more work for us and might not be worth it, but the benefit to the user is they wouldn't have to know our markup, and if the markup changed, we'd still have a consistent interface. I'm thinking like...
```js
alert.alertElement.focus()
```
or
```js
textField.inputElement.focus()
```
(using focus as an arbitrary example of something you'd want to do with that element)